themes/vue/source/css/_common.styl

```diff
@@ -57,7 +57,7 @@ a.button
   transition: all .15s ease
   box-sizing: border-box
   border: 1px solid lighten($green, 8%)
-  &.icon
+  &.has-icon
```

Comment:
styling is broken with this change - need to update the markup accordingly

Reply:
Updated this in the html